common/service/pldm/pldm_monitor.c

```diff
@@ -670,6 +672,56 @@ void set_effecter_state_gpio_handler(const uint8_t *buf, uint16_t len, uint8_t *
 	}
 }
 
+void pldm_spi_reinit(const char* spi_dev_str, const uint8_t *buf, uint16_t len, uint8_t *resp,
+				     uint16_t *resp_len)
+{
+	CHECK_NULL_ARG(buf);
+	CHECK_NULL_ARG(resp);
+	CHECK_NULL_ARG(resp_len);
+
+	uint8_t *completion_code_p = resp;
+	*resp_len = 1;
+
+	struct pldm_set_state_effecter_states_req *req_p =
+		(struct pldm_set_state_effecter_states_req *)buf;
+
+	if (req_p->composite_effecter_count !=
+	    PLDM_PLATFORM_OEM_SPI_REINIT_EFFECTER_STATE_FIELD_COUNT) {
+		LOG_ERR("Unsupported SPI reinit effecter count, (%d)",
+			req_p->composite_effecter_count);
+		*completion_code_p = PLDM_ERROR_INVALID_DATA;
+		return;
+	}
+
+	set_effecter_state_field_t *reinit_spi_state = &req_p->field[0];
+
+    switch (reinit_spi_state->set_request) {
```

Comment:
It seems like we're getting some cases where clang-format isn't being ran.  Please make sure it is ran on each commit.